api/v1alpha1/kubeadmbootstrapconfig_types.go

```diff
@@ -42,8 +30,8 @@ type KubeadmBootstrapConfigSpec struct {
 
 // KubeadmBootstrapConfigStatus defines the observed state of KubeadmBootstrapConfig
 type KubeadmBootstrapConfigStatus struct {
-	// Phase is the state of the KubeadmBootstrapConfig object
-	Phase Phase `json:"phase"`
+	// Ready indicates the BootstrapData field is ready to be consumed
+	Ready *bool `json:"phase,omitempty"`
```

Comment:
Based on recent changes to capi should probably be `bool` instead of `*bool`

Reply:
haha damn, that moves fast. i checked earlier today and it was `*bool` everywhere! will update, thanks for the 👀